.travis.yml

```diff
@@ -6,7 +6,6 @@ python:
 env:
   - TOX_ENV=py35-django-18
   - TOX_ENV=py34-django-18
-  - TOX_ENV=py33-django-18
```

Comment:
Why is this moved to the matrix below?

Reply:
Python 3.3 has reached end-of-life, you have to install it manually now.

Reply:
Yeah, got it after stumping into travis errors. Thanks!